targets/pna_nic/tests/utils.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2024 Marvell Technology, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+/*
+ * Rupesh Chiluka (rchiluka@marvell.com)
+ *
+ */
+
+#ifndef PNA_NIC_TESTS_UTILS_H_
+#define PNA_NIC_TESTS_UTILS_H_
+
+#include <vector>
+#include <mutex>
+#include <condition_variable>
+
+class PacketInReceiver {
```

Comment:
This is copied verbatim from targets/simple_switch
Can we have common test utility classes / functions in a shared folder under `targets/`? Maybe as `targets/test_utils`?

Reply:
Ok. Will create common files.